packages/perseus/src/widgets/plotter/plotter.tsx

```diff
@@ -46,6 +47,8 @@ type State = {
     categoryHeights: Record<string, number>;
 };
 
+const plotterDefaults = CoreWidgetRegistry.getDefaultWidgetOptions("plotter");
```

Comment:
So we have several ways to default data:

- The parser defaults data
- We have `getDefaultWidgetOptions`
- We have `DefaultProps`

For some reason Plotter had multiple values for the same defaults. I'm talking with Ben on how we can consolidate all of these defaults.